examples/decoding/plot_receptive_field.py

```diff
@@ -82,11 +82,11 @@
 mne.viz.tight_layout()
 
 ###############################################################################
-# Create and fit a receptive field model (a.k.a. forward model)
-# -------------------------------------------------------------
+# Create and fit a receptive field model
+# --------------------------------------
 #
-# We will construct a model to find the linear relationship between the EEG
-# signal and a time-delayed version of the speech envelope. This allows
+# We will construct an encoding model to find the linear relationship between
+# the EEG signal and a time-delayed version of the speech envelope. This allows
```

Comment:
minor point: can you reverse `EEG signal` and `time-delayed version...`, so that it follows the structure of the model as well?